### PR TITLE
[14.0][MIG] printer_zpl2

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Standard migration.

@ForgeFlow

### CODE REVIEW

File: base_report_to_printer/models/printing_server.py [hunk 1]
```diff
@@ -36,6 +36,13 @@ def _open_connection(self, raise_on_error=False):
         self.ensure_one()
         connection = False
         try:
+            # Sometimes connecting to printer servers outside of the local network
+            # can result in a weird error "cups.IPPError: (1030, 'The printer
+            # or class does not exist.')".
+            # An explicit call to `setServer` and `setPort` fixed the issue.
+            # (see https://github.com/OpenPrinting/pycups/issues/30)
+            cups.setServer(self.address)
+            cups.setPort(self.port)
```
Comment (etobella) [inline base_report_to_printer/models/printing_server.py:45]: Might we have consequences if you have more than one server?

Comment (LoisRForgeFlow) [reply base_report_to_printer/models/printing_server.py:45]: Each time you print a new file from a different server, a new connection will be spawned, so I don't see any problem

Comment (LoisRForgeFlow) [reply base_report_to_printer/models/printing_server.py:45]: Tested, no issue at all.

**Review by MiquelRForgeFlow — APPROVED** [review]
code LGTM

**Review by etobella — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
Wrong command, aborting and trying again:

/ocabot merge nobump

**Pull request merged**